src/pages/Profile/Modal/SubmitModal.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect, useContext } from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/OrangeMGE-ITP/SmartQ-ITP-Front/pullRequest?prid=8748018)
```suggestion
import React, { useState, useEffect, useContext } from "react";
```